build.py

```diff
@@ -168,20 +168,24 @@ def get_sources(task_name):
 
 
 def main(argv):
+
+
     args = getopts(argv)
 
     def object_of(source_filename, extension = ".o"):
         return args.build_dir / (source_filename.name + extension)
 
+    hjson_conf_file = "app."+args.product+".hjson"
```

Comment:
Please try to separate operators with whitespaces.
Alternatively, you can use python formats (which I find more readable):

```
hjson_conf_file = f"app.{args.product}.hjson"
```